.github/workflows/server-charm-release-edge.yml

```diff
@@ -4,24 +4,60 @@ on:
   push:
     branches:
       - main
+      - consolidate-server-workflows
```

Comment:
probably not needed anymore?

Reply:
oops, I thought I had reverted that already. Done now, thanks!